code/game/objects/structures/dropship_equipment.dm

```diff
@@ -762,7 +762,40 @@
 
 //////////////// OTHER EQUIPMENT /////////////////
 
+/obj/structure/dropship_equipment/operatingtable
+	name = "Dropship Operating Table Deployment System"
+	desc = "Used for advanced medical procedures. Fits on the crewserved weapon attach points of dropships. You need a powerloader to lift it."
+	equip_category = DROPSHIP_CREW_WEAPON
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "table2-idle"
+	point_cost = 300
+	var/obj/machinery/optable/deployed_table
+
+/obj/structure/dropship_equipment/operatingtable/Initialize()
+	. = ..()
+	if(!deployed_table)
+		deployed_table = new(src)
+
+/obj/structure/dropship_equipment/operatingtable/examine(mob/user)
+	. = ..()
+	if(!deployed_table)
+		to_chat(user, "Its table is broken.")
+		
+/obj/structure/dropship_equipment/operatingtable/Destroy()
+	QDEL_NULL(deployed_table)
+	return ..()
 
+/obj/structure/dropship_equipment/operatingtable/update_equipment()
+	if(!deployed_table)
+		return
+	if(ship_base)
+		deployed_table.layer = ABOVE_OBJ_LAYER
+		deployed_table.loc = loc
+		icon_state = "table2-idle"
+	else
+		deployed_table.layer = ABOVE_OBJ_LAYER
+		deployed_table.loc = src
+		icon_state = "table2-idle"
```

Comment:
What is `ship_base` doing for you here? From what i can tell you could just remove the check and do the same thing each time.
```suggestion
	deployed_table.layer = ABOVE_OBJ_LAYER
	deployed_table.loc = loc
	icon_state = "table2-idle"
```

Reply:
I have no clue, I'm just doing what hugh suggested.

Reply:
@Hughgent

Reply:
This is copied 'directly' from the mg nest, so it's replicated there as well I presume.

Reply:
the mg nest was updating their icon based on if it was attached or not.
In this case it looks like we are not.

I'd take my suggestion and then i think we'd be good to merge.